openghg/standardise/surface/_icos.py

```diff
@@ -217,6 +217,7 @@ def _read_data_large_header(
         "network": network,
         "instrument": instrument,
     }
+    attributes = {"inlet_height_magl": metadata["inlet_height_magl"], "data_owner": "See data owner email"}
```

Comment:
In the "small header" function, it says data owner is not set, but it says "see data owner email". In the tests, the expected value for the "larger header" version is the list of emails. I "see data owner email" converted to the list of emails at some point? I guess it would make more sense to me to keep the data owner emails in the `data_owner_email` attribute and leave this message in `data_owner` (or make `data_owner` be "NOT_SET")